src/app/helpers/helpers.component.ts

```diff
@@ -237,6 +237,14 @@ export class HelpersComponent implements OnInit, OnDestroy {
       column: helpers.eventComment,
       width: '120'
     });
+
+    if (this.context.isAllowed(Roles.distCenterAdmin) && this.settings.isSytemForMlt()) {
```

Comment:
not sure that a dist center admin can allow independent - I think it should be admin